Sources/FormattingHelpers.swift

```diff
@@ -403,7 +403,10 @@ extension Formatter {
                         }
                     }
 
-                    if let notWhitespace = index(of: .nonSpaceOrCommentOrLinebreak, before: openBracket) {
+                    // search backward and replace whitespace with a single " "
+                    // if we find a line comment (// ...) do not make this change
```

Comment:
Looks like this supports both line comments and block comments based on the test cases

```suggestion
                    // if we find a comment do not make this change
```